### PR TITLE
add installation instruction for Gentoo

### PR DESCRIPTION
**Pull request by dm9pZCAq**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -117,7 +117,8 @@ list:
...
 | https://getfedora.org/[Fedora Linux]                                                                                                                      | https://dnf.readthedocs.io/en/latest/[DNF]                                   | https://src.fedoraproject.org/rpms/rust-just[just]                                         | `dnf install just`
-|=================================================================================================================================================================================================================================================================================================================================================================
+| https://www.gentoo.org/[Gentoo Linux], https://github.com/gentoo-mirror/dm9pZCAq[dm9pZCAq overlay]                                                        | https://wiki.gentoo.org/wiki/Portage[Portage]                                | https://github.com/gentoo-mirror/dm9pZCAq/tree/master/sys-devel/just[sys-devel/just]       | `eselect repository enable dm9pZCAq && emerge --sync dm9pZCAq && emerge sys-devel/just`
```
Comment (casey) [inline README.adoc]: How about something like this:

```suggestion
| https://www.gentoo.org/[Gentoo Linux] | https://wiki.gentoo.org/wiki/Portage[Portage]                                | https://github.com/gentoo-mirror/dm9pZCAq/tree/master/sys-devel/just[dm9pZCAq overlay: sys-devel/just]       | `eselect repository enable dm9pZCAq && emerge --sync dm9pZCAq && emerge sys-devel/just`
```

**Review by casey — COMMENTED** [review]
Nice, thank you very much for packaging `just` for Gentoo! One minor comment.

Unrelated to this PR, do you know what the process would be for getting `just` into the official Gentoo repositories?

**Comment by dm9pZCAq** [comment]
> do you know what the process would be for getting `just` into the official Gentoo repositories?

right now Gentoo don't accept new packages like this, due to lack of maintainers

but Gentoo have [GURU](https://wiki.gentoo.org/wiki/Project:GURU) overlay which accepts any packages, it's like `aur` in Arch, but there are many outdated and bad written `ebuilds`, so i have own overlay with "good written" and up-to-date `ebuilds`

i'm watching for updates through [rss](https://github.com/casey/just/releases.atom) (about once every two or three days)

**Comment by casey** [comment]
> > do you know what the process would be for getting `just` into the official Gentoo repositories?
> 
> right now Gentoo don't accept new packages like this, due to lack of maintainers

Ahh, interesting.

> but Gentoo have [GURU](https://wiki.gentoo.org/wiki/Project:GURU) overlay which accepts any packages, it's like `aur` in Arch, but there are many outdated and bad written `ebuilds`, so i have own overlay with "good written" and up-to-date `ebuilds`

Nice!

> i'm watching for updates through [rss](https://github.com/casey/just/releases.atom) (about once every two or three days)

Oh cool, I didn't even know about the RSS feed.

**Comment by casey** [comment]
Merged, thank you very much for packing `just`, and for the PR!

**Comment by dm9pZCAq** [comment]
Hi, @casey
[GURU overlay now contains](https://gpo.zugaina.org/Overlays/guru/sys-devel/just) your package and it shows up in the [repology](https://repology.org/project/just/versions) in README, so you can remove my overlay from table in README

**Pull request merged**